entity-service-api/src/main/proto/org/hypertrace/entity/query/service/v1/entity_query_request.proto

```diff
@@ -25,6 +25,19 @@ message EntityUpdateRequest {
   repeated Expression selection = 4;
 }
 
+message BulkUpdateRequest {
+  string entityType = 1;
+  map<string, EntityUpdateInfo> bulkUpdateMap = 2;
+  message EntityUpdateInfo {
+    repeated UpdateInfo updateInfo = 1;
```

Comment:
Once we move out `repeated Expression selection = 2;` from `UpdateInfo`, this can be changed to `repeated UpdateOperation updateOperation = 1;` and then completely get rid of `UpdateInfo`